docs/data/material/discover-more/showcase/appList.js

```diff
@@ -519,6 +519,15 @@ const appList = [
     stars: 18500,
     dateAdded: '2021-05-06',
   },
+  {
+    title: 'Saleor Store Dashboard',
+    description: 'Dashboard application for Saleor headless e-commerce platform',
+    image: 'saleor.jpg',
+    link: 'https://demo.saleor.io/dashboard/',
+    source: 'https://github.com/saleor/saleor-dashboard',
+    stars: 412,
```

Comment:
```suggestion
    stars: 15079,
```
Decided to put main repo stars count since Dashboard is crucial part of the platform, but just developed in separate repository.